src/components/ArticleList.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from "react";
+import { fetchArticles } from "../api.js";
+
+const ArticleList = ({ articles, setArticles }) => {
+
+    useEffect(()=>{
+        fetchArticles(setArticles).then((articlesList) => {
+            setArticles(articlesList)
+        })
+    })
+
+  return (
+    <section>
+      {" "}
+      <ul className="list-flex-container">
+        {" "}
+        {articles.map((article) => {
+          return (
+            <li key={article.article_id} className="article-flex-item">
+              <img
+                className="article-list-article-img"
+                src={article.article_img_url}
+                alt={article.title}
+              />
+              <p>{article.title} </p>
+            </li>
```

Comment:
This is clearly laid out, well done. If you have time it would be great to see more information about the articles, author/comment count etc. Date is especially important for if you eventually sort by date. You might have already considered adding this later, forgive me